h2o-py/tests/testdir_algos/glm/pyunit_benign_glm.py

```diff
@@ -7,11 +7,10 @@
 
 def test_benign():
     training_data = h2o.import_file(pyunit_utils.locate("smalldata/logreg/benign.csv"))
-
+    training_data[3] = training_data[3].asfactor()
```

Comment:
This fix should also be in a different PR.

Reply:
Done.